app/services/analytics_events.rb

```diff
@@ -179,6 +179,14 @@ def email_and_password_auth(
     )
   end
 
+  # @deprecated
+  # A user has downloaded their personal key. This event is no longer emitted.
```

Comment:
YAGNI?

Or is this just so that we have a reference in the analytics documentation for historical lookups?

Reply:
Yes, I figured it would be good to have a reference. And if we choose to implement frontend logging, we can link these two as "previous event names" for that or something